hummingbot/connector/exchange/bitfinex/bitfinex_exchange.pyx

```diff
@@ -10,7 +10,7 @@ from typing import Any, AsyncIterable, Dict, List, Optional
 import aiohttp
 from libc.stdint cimport int64_t
 from libcpp cimport bool
```

Comment:
The recommended practice is to do `from libcpp cimport bool as cppbool`, not import it as a Python name. Refer to `paper_trade_exchange.pyx` for how it's done.